### PR TITLE
Blog post now includes Author (like on front page)

### PR DESCRIPTION
**Pull request by jonnyarnold**

### CODE REVIEW

**Comment by reevoo-samuel** [comment]
*If you're new, read the [guidelines](https://github.com/reevoo/guidelines/blob/master/pull_requests.md).*

Good code is **tested**, **designed for change**, **easy to understand** and **able to cope when
things go wrong**.

Good pull requests are **descriptive**, **small** and **short-lived**.

**For the Author**
- [ ] Description explains what you have done and why.
- [ ] Tests have been written (or you have explained why you don't need
tests).
- [ ] Documentation/comments have been added/updated.

**For the Reviewer**
- [ ] The code is well-designed and able to cope with change.
- [ ] The code is easy to understand.
- [ ] Tests cover happy and unhappy paths in the code.
- [ ] Tests are green on CI (or pulled/tested on your machine).

**Comment by lpil** [comment]
You could have a `_data/authors.yml`, and pull the author details out of there if you want to keep it all in one place

```yml
lpil:
  name: Louis Pilfold
  site: http://github.com/lpil
jonny:
  name: Jonny Arnold
  site: https://twitter.com/jonnyarnold89
```

```
{% assign author = site.data.authors[page.author] %}
{{ author.name }}
{{ author.site }}
```

**Comment by jonnyarnold** [comment]
We could; this is just keeping the display of Blog Summaries consistent between the front page and the blog page.

File: blog.html [hunk 1]
```diff
@@ -12,8 +12,18 @@
       <div class="post">
         <header class="post-header">
           <a href="{{ post.url }}"><h1>{{ post.title }}</h1></a>
-
-          <p class="meta">{{ post.date | date: "%b %-d, %Y" }}</p>
+          <p class="meta">{{ post.date | date: "%b %-d, %Y" }}
+            {% if post.author %}
+              -
+              {% if post.github %}
+                <a target="_blank" href="//github.com/{{ post.github }}">
```
Comment (davidsevcik) [inline blog.html]: `target="blank"` why?? Let the user decide hot to open a link.

**Comment by davidsevcik** [comment]
:+1:

**Pull request merged**